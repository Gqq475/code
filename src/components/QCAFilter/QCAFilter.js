import React from 'react'
import classes from './QCAFilter.scss'

type Props = {

};
export class QCAFilter extends React.Component {
  props: Props;

  render () {
    return (
      <div className={classes['QCAFilter-container']}>
        QCAFilter
      </div>
    )
  }
}

export default QCAFilter
