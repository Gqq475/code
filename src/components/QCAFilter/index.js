import QCAFilter from './QCAFilter'
export default QCAFilter
