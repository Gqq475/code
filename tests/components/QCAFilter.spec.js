import React from 'react'
import QCAFilter from 'components/QCAFilter/QCAFilter'

describe('(Component) QCAFilter', () => {
  it('render normally', () => {
    const wrapper = shallow(
      <QCAFilter />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
