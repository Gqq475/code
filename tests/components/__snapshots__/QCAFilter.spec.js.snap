// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) QCAFilter render normally 1`] = `
<div
  className="QCAFilter-container"
>
  QCAFilter
</div>
`;
